devito/interfaces.py

```diff
@@ -24,6 +24,16 @@
 # manipulation with the correct shapes, pointers, etc.
 _SymbolCache = {}
 
+# This global stores whether newly allocated memory should be initialised
+# using Devito's own JIT engine. It will otherwise be initialised using
+# numpy.
+Global_First_Touch = None
```

Comment:
This is already in ``configuration``, why having a global here would be any useful?